app/src/main/java/org/gdg/frisbee/android/eventseries/GdgEventListFragment.java

```diff
@@ -37,58 +41,33 @@ EventAdapter createEventAdapter() {
     }
 
     @Override
-    void fetchEvents() {
-        final DateTime now = DateTime.now();
-        DateTime start = now.minusMonths(2).dayOfMonth().withMinimumValue();
```

Comment:
Then, this is a behavior change. I am not sure about this. 
In the old behavior, the top event was always the next event. Now it is the latest event in the future.